doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -102,9 +103,17 @@ Complete the following steps to configure the modem trace level at compile time:
 The application can use the :c:func:`nrf_modem_lib_trace_level_set` function to set the desired trace level.
 Passing ``NRF_MODEM_LIB_TRACE_LEVEL_OFF`` to the :c:func:`nrf_modem_lib_trace_level_set` function disables trace output.
 
+.. note::
+   If the modem is shut down gracefully before the next boot (by using ``AT+CFUN=0``), it stores the current trace level.
```

Comment:
Saying that `AT+CFUN=0` shuts down the modem is a bit misleading and I would avoid this terminology here. The modem is powered on and some functionality is available after `AT+CFUN=0`. 
I would just say that the trace level is saved upon `AT+CFUN=0`.

Reply:
True, this was taken from `nrf91_features.rst`. I will update this.